docs/ECLLanguageReference/ECLR_mods/WkFlo-STORED.xml

```diff
@@ -51,6 +55,41 @@
             option is required when using STORED in a SOAP-enabled MACRO and
             the expected input is a dataset (such as tns:xmlDataset).</entry>
           </row>
+
+          <row>
+            <entry><emphasis role="bold">FORMAT</emphasis></entry>
+
+            <entry>Optional. FORMAT specifies options for formatting the field
+            on a Web form in WsECL.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">FIELDWIDTH</emphasis></entry>
+
+            <entry>Optional. FIELDWIDTH specifies the width of the input box
+            on a Web form in WsECL.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>widthvalue</emphasis></entry>
+
+            <entry>An integer expression defining the width (number of
+            characters) of the input box </entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">SEQUENCE</emphasis></entry>
+
+            <entry>Optional. SEQUENCE specifies field ordering on a Web form
+            in WsECL.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>sequencevalue</emphasis></entry>
+
+            <entry>An integer expression defining the sequential location of
+            the input box.</entry>
```

Comment:
You may want to mention that these can be sparse values... i.e. 1001, 2033, 2077.  This can be useful for managing the insertion of new inputs over time.